app.js

```diff
@@ -42,3 +45,20 @@ function hideCartHandler() {
   cart.style.transform = "translateX(100%)";
   cartOverlay.style.visibility = "hidden";
 }
+
+for (let i = 0; i < allChevronUpBtns.length; i++)
+  allChevronUpBtns[i].addEventListener("click", allChevronUpBtnsHandler);
+
+function allChevronUpBtnsHandler(event) {
+  event.target.nextElementSibling.innerHTML =
+    parseInt(event.target.nextElementSibling.innerHTML) + 1;
+}
+
+for (let i = 0; i < allChevronDownBtns.length; i++)
+  allChevronDownBtns[i].addEventListener("click", allChevronDownBtnsHandler);
+
+function allChevronDownBtnsHandler(event) {
+  if (event.target.previousElementSibling.innerHTML > 0)
```

Comment:
Now that I think about it, 0 amount for item doesn't make sense. I mean 0 item actually means that item is removed. So, let's make it 1 as the smallest possible amount.

Reply:
Thanks , I did not notice it but i have set it to 1 now.